README.md

```diff
@@ -1,9 +1,7 @@
-import useBaseUrl from "@docusaurus/useBaseUrl";
-
 # Amazon Connect Salesforce CTI Adapter Setup and Installation Guide
 
 <p align="center">
-  <img src={useBaseUrl('/static/img/lightning/image1.png')} />
+  <img src="./static/img/lightning/image1.png" />
```

Comment:
I think we should also add a link to the site somewhere here

Reply:
addressed!